api_tests/registries_moderation/test_submissions.py

```diff
@@ -236,3 +312,239 @@ def test_registries_moderation_permission_log(self, app, registration_log_url, r
 
         resp = app.get(registration_log_url, auth=moderator.auth)
         assert resp.status_code == 200
+
+    @pytest.mark.enable_quickfiles_creation
+    def test_registries_moderation_post_accept(self, app, registration, moderator, registration_actions_url, actions_payload_base, reg_creator):
+        registration.require_approval(user=registration.creator)
+
+        actions_payload_base['data']['attributes']['trigger'] = RegistrationModerationTriggers.SUBMIT.db_name
+        actions_payload_base['data']['attributes']['comment'] = 'Submitting registration'
+        actions_payload_base['data']['relationships']['target']['data']['id'] = registration._id
+
+        resp = app.post_json_api(registration_actions_url, actions_payload_base, auth=reg_creator.auth)
+        assert resp.status_code == 201
+        registration.refresh_from_db()
+        assert registration.moderation_state == RegistrationModerationStates.PENDING.db_name
```

Comment:
since we don't need to support the SUBMIT endpoint, you can skip over this whole mess by using the registration.registration_approval.accept() here and elsewhere